app/src/components/governance/PageProposal.vue

```diff
@@ -300,35 +309,13 @@ export default {
           )
         ) {
           this.found = true
+          this.loaded = true
+          return data.proposals.find(
```

Comment:
instead query for a single proposal from the API?

Reply:
the problem is there is no way to query for single proposals in Polkadot apparently

Reply:
and this query was here already, `proposals`, so it was kind of duplicating code

Reply:
we don't need to query for all proposals when just opening the proposal page right?

Reply:
so we can move the logic into the API even if we can not query for just one proposal in Polkadot.

Reply:
We should probably cash the proposals as well in store but in another PR

Reply:
OK, gotcha